msgraph/groups_test.go

```diff
@@ -65,6 +65,16 @@ func TestGroupsClient(t *testing.T) {
 	testGroupsClient_AddMembers(t, c, group)
 	testGroupsClient_RemoveMembers(t, c, *group.ID(), &([]string{c.Claims.ObjectId}))
 
+	testGroupsClient_AssignLicense(t, c, *group.ID(), &[]msgraph.GroupAssignedLicense{
+		{
+			DisabledPlans: &[]string{},
+			SkuId:         utils.StringPtr("90d8b3f8-712e-4f7b-aa1e-62e7ae6cbe96"), // SMB_APPS
```

Comment:
> GroupsClient.AssignLicense(): GroupsClient.BaseClient.Post(): unexpected status 400 with OData error: Request_BadRequest: License 90d8b3f8-712e-4f7b-aa1e-62e7ae6cbe96 does not correspond to a valid company License.

Please let me know the appropriate skusid as I did not know the proper ID to use for the test.

ref: https://learn.microsoft.com/ja-jp/entra/identity/users/licensing-service-plan-reference